src/layers/sds/epr/creators.py

```diff
@@ -94,14 +94,12 @@ def create_as_device(
     as_device_data: QuestionnaireResponse,
     message_sets_id: str,
     additional_interactions_id: str,
-    as_tags: list[dict],
 ) -> Device:
     as_device = product.create_device(
         name=EprNameTemplate.AS_DEVICE.format(party_key=party_key, asid=asid)
     )
     as_device.add_key(key_type=DeviceKeyType.ACCREDITED_SYSTEM_ID, key_value=asid)
     as_device.add_questionnaire_response(as_device_data)
-    as_device.add_tags(tags=as_tags)
```

Comment:
don't think we want to remove tags do we?

Reply:
Ah I was following the pattern for mhs, as they were removed there, and I thought they'd be captured in the tags ticket. But do AS Devices still need the tags of the sds fhir query params? And mhs have no tags then?

Reply:
yeah AS tags are still needed for SDS FHIR API - but there are fewer of them, and MHS devices don't have tags any more
=